src/components/demos/read/Modal.js

```diff
@@ -37,36 +37,48 @@ const renderActions = ({ update, readDemo }) => {
   const btn = ({ cyPrefix = '' }) =>
     readDemo.theme.name === 'Morello'
       ? [
-          <Button
-            key={'read-demo-modal-btn-yes-1'}
-            data-cy={`${cyPrefix}modal-btn-yes-cheri`}
-            onClick={handleYes}
-          >
-            YES
+          <Button key={'read-demo-modal-btn-yes-1'} onClick={handleYes}>
+            <DemoText
```

Comment:
we should probably remove color from below because there's so many cases where the color isn't white for Monaco/black for OpenSans
```
    font: isCheri
      ? { fontFamily: 'OpenSans', color: '#000' }
      : { fontFamily: 'Monaco', color: '#fff' },
 ```

Reply:
https://digicatapult.atlassian.net/browse/DSBD-102